### DISCUSSION TITLE
is Bootstrap latest version is free?

### DISCUSSION THREAD

**Discussion by adimanjre** [twbs/bootstrap#37697] [Q&A] [answered]
Bootstrap has launched new version that is 5.2.2. is it free for personal as well as commercial uses? 
If any cost is involved, how much it is and how to pay?

**Comment by julien-deramond** [comment, marked as answer]
Hey @adimanjre. Bootstrap has a [MIT License](https://github.com/twbs/bootstrap/blob/main/LICENSE). I'll let you check the details of the license but as a summary Bootstrap library is free for personal and commercial uses.
